indexer/src/main.ts

```diff
@@ -34,15 +43,10 @@ import {
 // Eth
 import { Bloom, Trie } from "./deps.ts";
 
-const AUTH_TOKEN = Deno.env.get("APIBARA_AUTH_TOKEN") ?? "";
-const TRANSACTION_EXECUTED = hash.getSelectorFromName("transaction_executed");
-
-const STREAM_URL = Deno.env.get("STREAM_URL") ?? "http://localhost:7171";
-const STARTING_BLOCK = Number(Deno.env.get("STARTING_BLOCK")) ?? 0;
 if (!Number.isSafeInteger(STARTING_BLOCK) || STARTING_BLOCK < 0) {
   throw new Error("Invalid STARTING_BLOCK");
 }
-const SINK_TYPE = Deno.env.get("SINK_TYPE") ?? "console";
+
 if (SINK_TYPE !== "console" && SINK_TYPE !== "mongo") {
   throw new Error("Invalid SINK_TYPE");
 }
```

Comment:
Can we also migrate this? (everything related to constants/check of constants should be in the constant file ideally)